Assets/RTLTMPro/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ These are the release notes for the RTLTMPro UPM package.
 The github repository is: https://github.com/engage-xr/RTLTMPro
 The parent repository is: https://github.com/hk1ll3r/RTLTMPro
 
+# v3.4.7 (2024/08/14)
```

Comment:
Is it a good idea to modify the base repo versioning? I think we should probably keep it and just use our patch tags as usual. If they release a new version we'll have problems.

Reply:
I doubt we'll see a new version... They haven't made a release in 2 years.